modules/ROOT/pages/constraints/managing-constraints.adoc

```diff
@@ -0,0 +1,1598 @@
+| INTEGER
+
+| name
+| Name of the constraint (explicitly set by the user or automatically assigned). label:default-output[]
+| STRING
+
+| type
+| The ConstraintType of this constraint (`UNIQUENESS` (node uniqueness), `RELATIONSHIP_UNIQUENESS`, `NODE_PROPERTY_EXISTENCE`, `RELATIONSHIP_PROPERTY_EXISTENCE`, `NODE_PROPERTY_TYPE`, `RELATIONSHIP_PROPERTY_TYPE`, `NODE_KEY`, or `RELATIONSHIP_KEY`). label:default-output[]
+| STRING
+
+| entityType
+| Type of entities this constraint represents (nodes or relationship). label:default-output[]
+| STRING
+
+| labelsOrTypes
+| The labels or relationship types of this constraint. label:default-output[]
+| LIST<STRING>
+
+| properties
+| The properties of this constraint. label:default-output[]
+| LIST<STRING>
+
+| ownedIndex
+| The name of the index associated with the constraint or `null`, in case no index is associated with it. label:default-output[]
+| STRING
+
+| propertyType
+| The property type the property is restricted to for property type constraints, or `null` for the other constraints.
+label:default-output[] label:new[Introduced in 5.9]
+| STRING
+
+| options
+| The options passed to `CREATE` command, for the index associated to the constraint, or `null` if no index is associated with the constraint.
+| MAP
+
+| createStatement
+| Statement used to create the constraint.
+| STRING
+
+|===
+
+[[drop-constraint]]
+== DROP CONSTRAINT
+
+Constraints are dropped using the `DROP CONSTRAINT` command.
+For the full command syntax to drop constraints, see xref:constraints/syntax.adoc#drop-constraint[Syntax -> DROP CONSTRAINT].
+
+[NOTE]
+Dropping a constraint requires the link:{neo4j-docs-base-uri}/operations-manual/{page-version}/authentication-authorization/database-administration/#access-control-database-administration-constraints[`DROP CONSTRAINT` privilege].
+
+.Drop a constraint by name
+======
+
+A constraint can be dropped using the name with the `DROP CONSTRAINT constraint_name` command.
+It is the same command for all constraint types.
+The name of the constraint can be found using the xref::constraints/syntax.adoc#constraints-syntax-list[`SHOW CONSTRAINTS` command], given in the output column `name`.
+
+.Drop the constraint `book_isbn`
+[source, cypher]
+----
+DROP CONSTRAINT book_isbn
+----
+
+.Result
+[source, queryresult]
+----
+Removed 1 constraint.
+----
+
+======
+
+.Drop a constraint using a parameter label:new[Introduced in 5.16]
+======
+
+The constraint name can also be given as a parameter.
+
+.Parameters
+[source, parameters]
+----
+{
+  "name": "actor_fullname"
+}
+----
+
+.Drop a constraint with a parameterized name
+[source, cypher]
+----
+DROP CONSTRAINT $name
+----
+
+.Result
+[source, queryresult]
+----
+Removed 1 constraint.
+----
+
+======
+
+.Drop a non-existing constraint
+======
+
+If it is uncertain if any constraint with a given name exists and you want to drop it if it does but not get an error should it not, use `IF EXISTS`.
```

Comment:
this text does not say a notification gets returned like the create counterpart does, might be nice if it did (and when that was added, same time as the create one)